app/src/main/java/org/simple/clinic/addidtopatient/searchresults/AddIdPatientSearchResultsScreenEvents.kt

```diff
@@ -0,0 +1,14 @@
+package org.simple.clinic.addidtopatient.searchresults
+
+import org.simple.clinic.patient.PatientSearchResult
+import org.simple.clinic.widgets.UiEvent
+
+data class AddIdToPatientSearchResultsScreenCreated(val key: AddIdToPatientSearchResultsScreenKey) : UiEvent
```

Comment:
Please do not pass keys to the controllers. Keys are an implementation detail on the UI layer, and controllers do not need to know about keys. If you need to pass along all the information, in a key to the controller, please declare them as properties of the `ScreenCreated` event.